cartography/intel/azure/iam.py

```diff
@@ -99,12 +99,15 @@ def get_managed_identity_client(credentials: Credentials, subscription_id: str)
 
 
 @timeit
-def list_tenant_users(client: GraphRbacManagementClient, tenant_id: str) -> List[Dict]:
+def list_tenant_users(client: GraphServiceClient, tenant_id: str) -> List[Dict]:
+    """
+    List users using Microsoft Graph API
+    Docs: https://learn.microsoft.com/en-us/graph/api/user-list
+    """
     try:
-        users = client.users.list()
-
-        users = transform_users(client.users.list(), tenant_id)
+        users_response = client.users.get()
```

Comment:
is this valid api? shouldn't this be `list`? `get` is always for one object not a list